### DISCUSSION TITLE
Is there a way to use the API to post or patch a new record with a many-to-many / linked record relationship?

### DISCUSSION THREAD

**Discussion by 684efs3** [nocodb/nocodb#4019] [Q&A]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

I am using Python and have been successful in uploading a new record and updating a linked record relationship that is one-to-many (and thus has a foreign key in the table). However, I want to upload a record that has a linked record, with a many-to-many relationship. How do I do this?

### Desired Behavior

Clear API directions on how to update a many-to-many linked record relationship.

### Project Details

Node: v16.16.0
Arch: x64
Platform: darwin
Docker: false
Database: pg
ProjectOnRootDB: false
RootDB: sqlite3
PackageVersion: 0.96.4

**Comment by sam-lev-fd** [comment]
There's an API path to add a relation row:
POST /api/v1/db/data/v1/Site API/sites/{rowId}/{relationType}/{columnName}/{refRowId}

relationType for a many to many would be mm

**Comment by pranavxc** [comment]
The following answer will be helpful - https://github.com/nocodb/nocodb/discussions/3930#discussioncomment-3802275
